downstream/assemblies/dev-guide/core

```diff
@@ -0,0 +1 @@
+../../modules/core/
```

Comment:
Missing URL in prerequisites section in assemblies/core/assembly-understanding-ansible-concepts.adoc:

"* Ansible is installed. For information about installing Ansible, see <link>."
Are prerequisites necessary in an assembly that includes only concept modules?